research/delf/delf/python/training/README.md

```diff
@@ -179,6 +175,14 @@ python3 train.py \
   --delg_global_features
 ```
 
+### Hyperparameter Guidelines
+
+In order to improve the convergence of the training, the following
```

Comment:
just to clarify, here you mean that all other flags for train.py should use the default values, right? If so, maybe briefly mention it here.

Reply:
Yes, that is correct. I added a mention to clarify this. Thanks for picking this up!